### PR TITLE
Add common configuration and troubleshooting tasks

### PR DESCRIPTION
**Pull request by vsinghal13**
###### Description

This PR add common configuration and troubleshooting tasks which will help the customer 
to customize the solution as per their needs. 

Configs added: 
1. Excluding logs/metrics for certain namespace/container
2. Configure local fluent-bit source

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet [hunk 1]
```diff
@@ -4,149 +4,7 @@
     clusterName: "kubernetes"
   },
   sumologicCollector:: {
-    remoteWriteConfigs+: [
```
Comment (frankreno) [inline deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet]: Why did this file change? Something in the CI?

Comment (samjsong) [reply deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet]: weird, good catch Frank. @vsinghal13 can you take a look please?

Comment (vsinghal13) [reply deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet]: hmm.. looks like something in the CI. will take a look.

Comment (vsinghal13) [reply deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet]: The `build.sh` copies the remote write section from the `prometheus-overrides.yaml` to this file and then does a `git diff` to generate new files. But there is no change in the remote write config in this PR.

**Pull request merged**